README.md

```diff
@@ -24,6 +25,10 @@ We would build more detailed tutorials in the near future as the repository is u
 - Python 3.x (for complete project)
 - py4j library (for building importers only)
 
+## Installing Virtual Environments
+- [Anaconda](https://conda.io/docs/user-guide/install/index.html#regular-installation)
```

Comment:
Anaconda on windows comes with jupyter. Maybe worth seperating between windows and os, linux. So we can just say Anaconda for Windows and virtualenv (sudo pip install virtualenv) with jupyter (pip install jupyter)